GPU/Math3D.h

```diff
@@ -876,33 +882,44 @@ float vectorGetByIndex(__m128 v) {
 }
 #endif
 
-// v and vecOut must point to different memory.
-inline void Vec3ByMatrix43(float vecOut[3], const float v[3], const float m[12]) {
 #if defined(_M_SSE)
+inline __m128 MATH3D_CALL Vec3ByMatrix43(__m128 x, __m128 y, __m128 z, const float m[12]) {
```

Comment:
should comment here that x, y, z must contain those values filling out their respective SSE registers, not just once.